notificationworkerlambda/src/main/scala/com/gu/notifications/worker/Harvester.scala

```diff
@@ -62,17 +72,16 @@ trait HarvesterRequestHandler extends Logging {
   def queueShardedNotification(shardNotifications: Stream[IO, ShardedNotification]): Stream[IO, Unit] = {
     for {
       n <- shardNotifications
-      platform = n.platform
-      sink: Sink[IO, String] = platform match {
-        case Android => firebaseSink(n.notification, n.range)
-        case _root_.models.iOS => apnsSink(n.notification, n.range, platform)
-        case Newsstand => apnsSink(n.notification, n.range, platform)
-        case _ => throw new Exception("Unknown platform")
-      }
+      firebaseSink = firebaseSinkBuilder(n.notification, n.range)
+      newsstandSink = apnsSink(n.notification, n.range, Newsstand)
```

Comment:
Don't get why this is hardcoded to Newsstand

Reply:
Apple notifications are either Newsstand or iOS, so there are two sinks that might be used.
For some reason, Newsstand is its own platform (can't remember history).
Maybe this could be revisited to see what value it adds, I'm not sure it's exposed in metrics past this point. Can I raise a ticket for this?

Reply:
Yes  that is a good plan.